packages/components/input-tag/src/composables/use-input-tag-dom.ts

```diff
@@ -66,6 +76,50 @@ export function useInputTagDom({
     )
   })
 
+  const states = reactive(<InputTagStates>{
+    innerWidth: 0,
+    collapseItemWidth: 0,
+  })
+
+  const getGapWidth = () => {
+    if (!innerRef.value) return 0
+    const style = window.getComputedStyle(innerRef.value)
+    return Number.parseFloat(style.gap || '6px')
+  }
+
+  const getInnerWidth = () => {
+    if (!innerRef.value) return 0
+    states.innerWidth = Number.parseFloat(
+      window.getComputedStyle(innerRef.value).width
+    )
+  }
+
+  const resetCollapseItemWidth = () => {
+    if (!collapseItemRef.value) return 0
+    states.collapseItemWidth =
+      collapseItemRef.value.getBoundingClientRect().width
```

Comment:
```suggestion
    states.collapseItemWidth =
      collapseItemRef.value!.getBoundingClientRect().width
```

See https://github.com/vueuse/vueuse/blob/main/packages/core/useResizeObserver/index.ts#L80-L81